_tools/node_list_remaining_tests.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+import { walk } from "../fs/walk.ts";
+import { withoutAll } from "../collections/without_all.ts";
+
+// deno-lint-ignore no-explicit-any
+type Object = Record<string, any>;
+
+const encoder = new TextEncoder();
+
+const NODE_BASE_URL = "https://api.github.com/repos/nodejs/node";
+
+async function getNodeTestDirSHA(): Promise<string> {
+  const response = await fetch(NODE_BASE_URL + "/contents");
+  const body = await response.json();
+  return body
+    .find(({ name }: Object) => name === "test")
+    .sha;
+}
+
+async function getNodeTests(sha: string): Promise<string[]> {
+  const url = NODE_BASE_URL + "/git/trees/" + sha + "?recursive=1";
+  const response = await fetch(url);
+  const body = await response.json();
+
+  return body.tree
+    .filter(({ path }: Object) => path.endsWith(".js"))
```

Comment:
I think the items under `fixtures/` should be filtered out as they are not test cases. Also items under `common/` look like the testing utilities, not test cases.

Reply:
Ok, this now has me thinking. Which [Node test directories](https://github.com/nodejs/node/tree/main/test) do we want to ignore for the foreseeable future completely? I think `testpy` should also be ignored.

@cjihrig, do you have any additional thoughts?

Reply:
We can probably skip addons, async-hooks, cctest, doctool, embedding, fixtures, fuzzers, js-native-api, node-api, overlapped-checker, report, testpy, tick-processor, tools, v8-updates, wasi, and wpt. As new functionality such as Node-API is added here, we could enable those tests.

To be honest, I kind of feel like maintaining this might not be worth the work. Having a script that dumps out the missing tests could be useful, but I'm not sure that we need to maintain the list in the repo. Just my opinion - if others feel it's useful then by all means, add it.

Reply:
I plan to have an array containing the list of Node test directory names that should be ignored. In the future, I believe the extent of maintenance of this tool might include removing names from this array to have them to be tested. Nothing else beyond that, as far as I can see.

IMO, this is a cheap win - having a clear and up-to-date list of Node tests for contributors to pursue.